### PR TITLE
Added explicit details and solidity wrappers for precompiled contract at addresses 0x1 - 0x3

### PR DESCRIPTION
**Pull request by PaulElisha**
## Proposed changes

I added high-level details of precompiled contracts for easy comprehension.
I explicitly added solidity wrappers to precompiled contracts at the first four addresses for easy understanding/application.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [ x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [ x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x ] I have added the necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- PR #698

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by scott-klaytn** [comment]
@PaulElisha Please post "I have read the CLA Document and I hereby sign the CLA" in a comment as shown by CLA Assistant bot.

**Comment by PaulElisha** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/learn/computation/precompiled-contracts.md [hunk 1]
```diff
@@ -17,26 +22,53 @@ Precompiled contracts related hardfork changes can be found at the bottom of thi
...
+		assembly {
+			mstore(0, numberToHash) // store number in the zeroth memory word
+
+			let ok := staticcall(gas(), 2, 0, 32, 0, 32)
+			if iszero(ok) {
+				revert(0,0)
+			}
+			return(0, 32)
+		}
+	}
```
Comment (ayo-klaytn) [inline docs/learn/computation/precompiled-contracts.md:59]: make sure to align your code block properly

File: docs/learn/computation/precompiled-contracts.md [hunk 2]
```diff
@@ -1,8 +1,13 @@
 # Precompiled Contracts
 
-Klaytn provides several useful precompiled contracts. 
-These contracts are implemented in the platform itself as a native implementation.
+Klaytn provides several useful precompiled contracts, none of which are state changing.
```
Comment (ayo-klaytn) [inline docs/learn/computation/precompiled-contracts.md:3]: change to state-changing instead

File: docs/learn/computation/precompiled-contracts.md [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
+These contracts are implemented in the platform itself as a native implementation which means they are part of the Klaytn client specifications.
```
Comment (ayo-klaytn) [inline docs/learn/computation/precompiled-contracts.md:4]: add a comma after _native_ _implementation_

**Review by ayo-klaytn — APPROVED** [review]

**Review by scott-klaytn — APPROVED** [review]

**Pull request merged**